litmus-portal/frontend/src/views/ChaosWorkflows/BrowseAnalytics/WorkflowComparisonTable/styles.ts

```diff
@@ -0,0 +1,303 @@
+import { createMuiTheme, makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    height: '100%',
+    width: '100%',
+    flexDirection: 'column',
+    paddingBottom: theme.spacing(5),
+    marginLeft: theme.spacing(2),
+    paddingRight: theme.spacing(2),
+    overflow: 'hidden',
+  },
+
+  analyticsDiv: {
+    paddingRight: theme.spacing(3.75),
+  },
+
+  heading: {
+    marginTop: theme.spacing(7),
+    fontFamily: 'Ubuntu',
+    fontSize: '1.5rem',
+  },
+
+  description: {
+    width: '50rem',
+    marginTop: theme.spacing(3),
+    fontFamily: 'Ubuntu',
+    fontSize: '1rem',
+  },
+
+  tableFix: {
+    marginTop: theme.spacing(1),
+    width: '100%',
+  },
+
+  horizontalLine: {
+    marginBottom: theme.spacing(2),
+  },
+
+  headerSection: {
+    width: '100%',
+    display: 'flex',
+    flexDirection: 'row',
+    alignContent: 'center',
+    alignItems: 'center',
+    justifyContent: 'center',
+    border: '1px solid ',
+    height: '6rem',
+    borderColor: theme.palette.text.hint,
+    backgroundColor: theme.palette.common.white,
+  },
+
+  search: {
+    marginRight: 'auto',
+    marginLeft: theme.spacing(6.25),
+    borderBottom: '1px solid rgba(0, 0, 0, 0.1)',
+    width: '17.5rem',
+  },
+
+  calIcon: {
+    color: 'rgba(0, 0, 0, 0.4)',
```

Comment:
No we can't @Saranya-jena custom colours are inaccessible and throwing error in this component, after this release we will try to debug this, for a few components it happening not for all, i had earlier used custom colours but had to change because of the crash cc: @arkajyotiMukherjee

Reply:
did you try like this? `border: `1px solid ${theme.palette.grey[500]}`,` it should work

Reply:
try this @ishangupta-ds

Reply:
There will be a minor difference in the colour then @Saranya-jena i have picked colours from the figma so.. not sure if making changes to it will be good just for the sake of custom colours, i guess debugging the custom colours issue will be better and more appropriate...  @arkajyotiMukherjee and i have decided to debug that after release we will fix this later and use custom colours from theme.